docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source.md

```diff
@@ -116,6 +116,10 @@ When Sumo Logic detects an issue it is tracked by Health Events. The following t
 | ThirdPartyGeneric | Normally due to an error communicating with the third party service APIs. | Yes | The Source will retry for up to 90 minutes, after which retries will be attempted every 60 minutes. | ThirdPartyGenericError |
 | FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs. | Yes | The Source will retry for up to 90 minutes, after which retries will be attempted every 60 minutes. | FirstPartyGenericError |
 
+** You may receive an error: 
+
+"At least one receiver for the endpoint is created with epoch of '0', and so non-epoch receiver is not allowed. Either reconnect with a higher epoch, or make sure all epoch receivers are closed or disconnected." This is indicative of another solution pulling from the event hub and we are not able to pull from it. Please see this [document for reference](https://techcommunity.microsoft.com/t5/azure-paas-blog/eventhub-the-behavior-of-eph-sdk-behavior-of-epoch/ba-p/3888275). Please create a new eventhub instance that we are the primary consumer. 
```

Comment:
```suggestion
** You may receive an error: `At least one receiver for the endpoint is created with epoch of '0', and so non-epoch receiver is not allowed. Either reconnect with a higher epoch, or make sure all epoch receivers are closed or disconnected.` This indicates Sumo Logic cannot pull from the even hub because it detects another solution pulling from it. See [this Microsoft documentation](https://techcommunity.microsoft.com/t5/azure-paas-blog/eventhub-the-behavior-of-eph-sdk-behavior-of-epoch/ba-p/3888275) for more information. You'll need to create a new event hub instance where Sumo Logic is the primary consumer. 
```